+++ src/kernel/muen/policy/xml/vcpu_subject_base_hw.xml
@@ -9,7 +9,7 @@
 	</vmx>
 	<registers>
 		<gpr>
-			<rip>16#0020_0028#</rip>
+			<rip>16#0020_0078#</rip>
 			<rsp>16#0000#</rsp>
 		</gpr>
 	</registers>
